### ISSUE TITLE
Election target

### ISSUE THREAD

**Issue by verdverm** [sysgears/spinjs#38]
I've been working with https://github.com/chentsulin/electron-react-boilerplate lately ; pretty end-to-end like apollo-universal-starter-kit

What are your thoughts on adding something like this into SpinJS ?

**Comment by vlasenko** [comment]
@verdverm Yes, it would be great to add Electron target to SpinJS, we are going to add it, but we are busy with other tasks for now

**Comment by verdverm** [comment]
Cool, I've been learning more about webpack and such things, about time I took a look into SpinJS. I will probably try to prototype something before the month is out.
